provider/identity_provider_resource.go

```diff
@@ -392,11 +435,13 @@ func (r *IdentityProviderResource) Create(ctx context.Context,
 		Value: object.ID(),
 	}
 	htpasswdObject := object.Htpasswd()
+	gitlabObject := object.Gitlab()
 	ldapObject := object.LDAP()
 	openidObject := object.OpenID()
 	switch {
 	case htpasswdObject != nil:
 		// Nothing, there are no computed attributes for `htpasswd` identity providers.
+	case gitlabObject !=nil:
```

Comment:
Add a comment please that there is no need to do anything because there are no compute attributes for GitLab

Reply:
Done, comment added.